### ISSUE TITLE
Add tests for Telegram games

### ISSUE THREAD

**Issue by chrisonntag** [yagop/node-telegram-bot-api#212]
As of right now, the tests aren't covering all possible requests. The file test/index.js needs to be updated to also test functions in relation to Telegram games, e.g. `sendGame(...)`or `sendGameScore(...)`

As stated in CONTRIBUTING.md it's necessary to set some environment variables

``` bash
export TEST_TELEGRAM_TOKEN="123456:AAGoASDFJKnlj6uSQLIhjhBgOUY" # Your bot token
export TEST_USER_ID="66368736" # User wich the bot will send the messages (can be yours ID)
export TEST_GROUP_ID="-163889657" # A Telegram group which the bot have access
```

Once tests are created, just run `npm run test`.

**Comment by GochoMugo** [comment]
Some of these test cases are being addressed in PR #250!

**Comment by GochoMugo** [comment]
We are now testing TelegramBot#sendGame() and TelegramBot#sendGameScore().

Hope to receive PR(s) with more tests :smile:! We hoping to get our coverage to 100%.
